src/content/contribute/writing-a-loader.md

```diff
@@ -130,6 +121,25 @@ Take the case of rendering a template file with data specified via loader option
 
 T> The fact that loaders can be chained also means they don't necessarily have to output JavaScript. As long as the next loader in the chain can handle its output, the loader can return any type of module.
 
+### Data sharing
+
+As discussed earlier, webpack loaders can be chained. So, along with passing the content(source code) with the next chained loader, we can even share data to next loader. There are two ways to share data among loaders.
+
+__Using `this.callback` method of raw loaders__
```

Comment:
```suggestion
```

Reply:
@EugeneHlushko I think this line might be needed as there is one `callback` for pitch methods as well and they wont here pass the data to next loaders

Reply:
cc @EugeneHlushko , do we need to remove this ?

Reply:
This paragraph above is speaking about `this.callback` method of raw loaders. So does the bold part here, its duplication. Previously it was claiming two different ways of doing it and enlisting only one. Since we've done the rephrasing, it no more claims to have two ways of doing it, therefore this enlistment isnt required in my opinion. Unless you can add second way of doing it with an example